esp/templates/users/profiles/usercontact.html

```diff
@@ -74,7 +74,14 @@ <h4>Address:</h4>
 
 {{ form.address_postal }}
 
-<h4>Phone number{% if form.phone_day and form.phone_cell %}s{% endif %}:<font color="red">*</font></h4>
+<h4>Phone number{% if form.phone_day and form.phone_cell %}s{% endif %}:{% if profiletype != "student" or require_student_phonenum %}<font color="red">*</font>{% endif %}</h4>
+
+{% if profiletype == "student" and not require_student_phonenum %}
+<div class="alert">
+    <button type="button" class="close" data-dismiss="alert">x</button>
+    <strong>Students:</strong> Please enter <b>your</b> phone numbers below. If you do not have {% if form.phone_day %}a home phone {% endif %}{% if form.phone_day and form.phone_cell %}nor {% endif %} {% if form.phone_cell %}a personal cell phone{% endif %}, please leave this blank. You will enter your guardian and emergency contact phone numbers in the sections below.
+</div>
+{% endif %}
```

Comment:
I think we want the alert no matter what, but if the fields are required only show the first sentence. Otherwise, show the whole thing. Also, I think you want to get rid of the space here: `{% endif %} {% if form.phone_cell %}`